companion/src/firmwares/edgetx/yaml_modeldata.cpp

```diff
@@ -1163,8 +1162,13 @@ bool convert<ModelData>::decode(const Node& node, ModelData& rhs)
 
   qDebug() << "Settings version:" << modelSettingsVersion.toString();
 
-  if (modelSettingsVersion > SemanticVersion(VERSION))
-    qDebug() << "Warning: version not supported by Companion!";
+  if (modelSettingsVersion > SemanticVersion(VERSION)) {
+    QString prmpt = QCoreApplication::translate("YamlModelSettings", "Warning: Model settings file version %1 is not supported by this version of Companion!\n\nModel and radio settings may be corrupted if you continue.\n\nI acknowledge and accept the consequences.");
```

Comment:
Is it possible to display the model filename in question here?

Reply:
A present the best I could do is display the model name and add a TODO reminder.

Reply:
That would be great in the interim - better than a bunch of non-descript "something ain't right" errors. Just thinking back to the other todo about the error handling, maybe that is where the filename could get exposed as part of the reason for failing.

Reply:
One day.....